src/main/java/racingcar/model/RandomMoveStrategy.java

```diff
@@ -0,0 +1,18 @@
+package racingcar.model;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+public class RandomMoveStrategy implements MoveStrategy {
+
+    private static final int MIN = 0;
+    private static final int MAX = 9;
+    private static final int MOVE_THRESHOLD = 4;
+
+    @Override
+    public boolean canMove() {
+        int randomValue = Randoms.pickNumberInRange(MIN,MAX);
+
+        return randomValue >= MOVE_THRESHOLD;
+    }
```

Comment:
```suggestion
    @Override
    public boolean canMove() {
        int randomValue = Randoms.pickNumberInRange(MIN, MAX);

        return randomValue >= MOVE_THRESHOLD;
    }
```

, 이후에 한 칸 띄워주시면 더 좋을 것 같아요! 추가로 우테코 코드 스타일을 적용하신다면 저장하면 자동으로 컨벤션을 맞춰줍니다! 한번 적용해보시면 좋을 것 같아요!